flipperdevices/flipperzero-firmware
# CLI: log command argument (log level)

# What's new

- CLI `log` command now has a Log Level argument.

# Verification 

- Set Log Level in System Settings to None.
- Execute `log` command. Make sure no logs are output.
- Ctrl+C, execute `log trace`. Make sure the trace level logs are visible.
- Ctrl+C, execute `log`. Make sure no logs are output.

# Checklist (For Reviewer)
